extension/ts/background/workers/authorize.ts

```diff
@@ -0,0 +1,31 @@
+type AuthorizeParameters = boolean;
+type AuthorizeResponse = string;
+
+const authorize = (interactive: AuthorizeParameters): Promise<AuthorizeResponse> =>
```

Comment:
Yeah exactly. It will just fail with an error if they are not signed in. We `.catch` the error and then just don't show the filter stuff.

It would be nice to have a button there telling you that you can log in, but I don't want to finish that in time for the weekend (when I am targeting a 1.2.0 release)